README.md

```diff
@@ -215,6 +215,40 @@ app.get('/.well-known/acme-challenge/:acmeToken', function(req, res, next) {
 });
 ```
 
+### PHP Apps
+
+Add the following to `.well-known/acme-challenge/index.php`
+
+```php
+<?php
+$request = $_SERVER['REQUEST_URI'];
+if(preg_match('#^/.well-known/acme-challenge/#', $request) === 0) {
+    return;
+}
+
+if(isset($_ENV['ACME_KEY'])) die(trim($_ENV['ACME_KEY']));
+
+foreach($_ENV as $key => $value) {
+    if(preg_match('#^ACME_TOKEN_#', $key)) {
+        $number = str_replace('ACME_TOKEN_', '', $key);
+        die(trim($_ENV['ACME_KEY_'.$number]));
```

Comment:
Ahh that makes sense. So `acme-challenge/{KEY}` would == `$_ENV['ACME_KEY_{KEY}']`?